spec/v1/bundle.md

```diff
@@ -2,40 +2,61 @@
 
 Version: v1.0
 
-An attestation bundle is a collection of multiple attestations in a single
+An attestation Bundle is a collection of multiple attestations in a single
 file. This allows attestations from multiple different points in the software
 supply chain (e.g. Provenance, Code Review, Test Result, vuln scan, ...) to
 be grouped together, allowing users to make decisions based on all available
 information.
 
-**NOTE**: The bundle is not authenticated as a whole. Instead each individual
-attestation is authenticated using [DSSE]. As such, an attacker might be able
-to _delete_ an attestation without being detected. Predicates that follow
-[the monotonic principle] should not have any issues with this behavior.
+**NOTE**: The Bundle is not authenticated as a whole. Instead each individual
+attestation is authenticated using signature formats like [DSSE]. As such,
+an attacker might be able to _delete_ an attestation without being detected.
+Predicates that follow [the monotonic principle] should not have any issues
+with this behavior.
 
 ## Data structure
 
 Attestation Bundles use [JSON Lines] to store multiple attestations.
 
--   Each attestation within a bundle MAY have a different signing key,
+-   Each attestation within a Bundle MAY have a different signing key,
     `_type`, `subject`, and/or `predicateType`.
--   Each line SHOULD be an [Envelope]. Consumers MUST ignore unrecognized
-    lines.
+-   Each line within a Bundle SHOULD be an [Envelope].
```

Comment:
It's so the bundle can support other envelope types besides DSSE and allow for extensibility in the future.  This may be needed to support other existing formats like Sigstore's bundle, COSE, or any new bundle formats in the future.

@woodruffw and I discussed this some last week.

Reply:
I think that still calls for bumping this to a `MUST`. The text in this PR for [Envelope] has been updated to say:

> The RECOMMENDED format and protocol for Envelopes are defined per [DSSE v1.0]. Producers MAY use other signature methods and formats so long as they meet the [Bundle] data structure requirements.

So [Envelope] isn't just DSSE. Also, I'm still nervous about storing a sigstore bundle as an entry in the in-toto attestation bundle as that may not actually be a signature envelope if it only includes a raw signature + verification materials...

Reply:
Oh I missed that.  Previously Envelope was always a DSSE.

TBH it now sounds a bit circular?

Bundle says "every line must be an 'Envelope'"
Envelope says "an envelope can be anything as long as Bundle is OK with it"

What fields does an Envelope actually require?

Reply:
I say we move this discussion as part of #21.

Reply:
Sounds reasonable to move this out. I will also note that perhaps we can just point to ITE-5 and call it a day. It already lists the desired properties for a signature wrapper, and the in-toto 1.0 spec adheres to it.